onnx/tensor.h

```diff
@@ -0,0 +1,141 @@
+#pragma once
+
+#include "onnx/onnx_pb.h"
+
+namespace onnx {
+
+struct Tensor {
+private:
+  bool is_segment_;
+  int64_t segment_begin_;
+  int64_t segment_end_;
+  bool has_name_;
```

Comment:
I wonder whether we should remove it from here and instead use a map<> from name to value in initializers

Reply:
We can follow this issue here https://github.com/onnx/onnx/issues/438